pkg/eestream/encode.go

```diff
@@ -191,8 +191,6 @@ type encodedPiece struct {
 }
 
 func (ep *encodedPiece) Read(p []byte) (n int, err error) {
```

Comment:
Are there other places that might have this problem?

Reply:
The problem of making the trace file too big? Potentially yeah because recently some additional tracing was added that is tracing really fast functions but it's pretty difficult to open the trace file right now with these reads. Maybe after this PR we can sweep and do a cleanup PR once this PR trims the trace file size down to a size we can actually open and search.

Reply:
Would it be worth adding a comment like "tracing deemed unnecessary here" so that some future cleanup operation doesn't add it back again?

Reply:
maybe not unnecessary, but "Tracing here is too noisy"?

Reply:
Added a comment explaining why tracing was removed.